libraries/abstractions/platform/test/iot_test_platform_threads.c

```diff
@@ -71,7 +71,9 @@ TEST_TEAR_DOWN( UTIL_Platform_Threads )
 TEST_GROUP_RUNNER( UTIL_Platform_Threads )
 {
     RUN_TEST_CASE( UTIL_Platform_Threads, IotThreads_CreateDetachedThread );
-    RUN_TEST_CASE( UTIL_Platform_Threads, IotThreads_ThreadPriority );
+    #if ( configUSE_TRACE_FACILITY == 1 )
```

Comment:
Is this related to the lwIP upgrade?

Reply:
Yes.  The MediaTek board is quite constrained.  Disabling the trace facility permitted a successful compilation.